x/ibc/07-tendermint/types/header.go

```diff
@@ -37,10 +37,11 @@ func (h Header) GetHeight() exported.Height {
 	if h.Header == nil {
```

Comment:
remove the godoc TODO

Reply:
e3ebdde159046409b2785e85bf04a2c46449616a